src/gc/unix/gcenv.unix.cpp

```diff
@@ -517,6 +523,14 @@ void GCToOSInterface::GetMemoryStatus(uint32_t* memory_load, uint64_t* available
         *available_page_file = 0;
 }
 
+// Get size of an OS memory page
+void GCToOSInterface::GetPageSize()
```

Comment:
With move to GCToOSInterface::Inititialize, this will be moved to header and be inlined.